plugin/src/main/groovy/com/cookpad/android/licensetools/LicenseToolsPlugin.groovy

```diff
@@ -51,6 +51,7 @@ class LicenseToolsPlugin implements Plugin<Project> {
                         message.append("  url: ${libraryInfo.url ?: "#URL#"}\n")
                     }
                     project.logger.warn(message.toString().trim())
+                    appendToLicenseYaml(project, message.toString().trim())
```

Comment:
I think this is not bug fix, but new feature.
And, the feature should not be included into `checkLicenses` tasks.
It should split gradle task, so could you split PR for this feature? ( it's just an idea, the task would be called `updateLicenses`, do you think about? )